certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,133 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MOBSIM Send SMS",
+    "description": "Communicate with your customers in an assertive and direct way using the \"MOBSIM - Send SMS\" Connector. SMS - with high delivery and opening rates - makes it possible to send promotional and transactional campaigns, alerts, reminders, notifications, invites and more, allowing for a complete experience with personalized messages and the convenience of scheduled deliveries to mobile phone users.",
+    "version": "1.0",
+    "contact": {
+      "name": "Suporte Mobsim",
+      "url": "https://www.mobsim.com.br/",
+      "email": "contato@mobsim.com.br"
+      }
+  },
+  "host": "mobsim-api.com.br",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/sms": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
@jpjsantos it's not allowed to use default as the only response, please refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) when defining your responses.